src/GraphQL.Client/GraphQLHttpResponse.cs

```diff
@@ -19,6 +19,13 @@ public GraphQLHttpResponse(GraphQLResponse<T> response, HttpResponseHeaders resp
     public HttpStatusCode StatusCode { get; set; }
 }
 
+public interface IGraphQLHttpResponse : IGraphQLResponse
```

Comment:
This interface is needed to pass non-generic instance of response into `GraphQLHttpClientOptions.DisableAPQ`.

Reply:
I think you should put the content in there, too... then people can evaluate plaintext error messages...

**Always** having the raw content in there might be beneficial for general debugging, too, cause then people could cast their responses to `IGraphQLHttpResponse` and get the raw body of the response...

Reply:
Then I would put entire `HttpResponseMessage` instead of pulling discrete properties from it. Also note that content may be already unaccessable since stream was read in `SendHttpRequestAsync`.

Reply:
#559 , not a goal for this PR